webpack/common/ScalprumModule/ScalprumContext.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState, createContext, useCallback, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { initialize } from '@scalprum/core'
+import { ScalprumProvider } from '@scalprum/react-core';
+
+export const ScalprumContextWrapper = ({ children }) => {
+  const config = {
+    'vulnerability': {
+      name: 'vulnerability',
+      manifestLocation: `${window.location.origin}/pub/assets/apps/vulnerability/fed-mods.json`,
+      cdnPath: `${window.location.origin}/pub/assets/apps/vulnerability/`,
+    },
+  };
+
+  const scStore = useMemo(() => initialize(        {pluginSDKOptions: {
+          pluginLoaderOptions: {
+            transformPluginManifest: manifest => {
+              if (
+                manifest.baseURL === 'auto' &&
+                config[manifest.name]?.cdnPath
+              ) {
+                const _cdnPath = config[manifest.name]?.cdnPath;
+                return {
+                  ...manifest,
+                  baseURL: _cdnPath,
+                  loadScripts: manifest.loadScripts.map(
+                    script => `${_cdnPath}${script}`
+                  ),
+                };
+              }
+              return manifest;
+            },
+          },
+        },
+        api: {
+          chrome: {
+            isBeta: () => false,
+            on: () => {},
+            auth: {
+              getUser: () => Promise.resolve(mockUser),
+            },
+          },
+        },
+        config}), [])
+
+  const mockUser = {
+    entitlements: {},
+    identity: {
+      account_number: 'string',
+      org_id: 'string',
+      internal: {
+        org_id: 'string',
+        account_id: 'string',
+      },
+      type: 'string',
+      user: {
+        username: 'string',
+        email: 'string',
+        first_name: 'string',
+        last_name: 'string',
+        is_active: 'boolean',
+        is_internal: 'boolean',
+        is_org_admin: 'boolean',
+        locale: 'string',
+      },
+    },
+  };
+  return (
+      <ScalprumProvider
+        scalprum={scStore}
```

Comment:
https://github.com/scalprum/scaffolding/blob/8671c64a31e7c5f7ac9675fa95067cc60faf933c/packages/react-core/src/scalprum-provider.tsx#L19
ScalprumProvider doesnt have a scalprum props
why not use the props as in https://github.com/theforeman/foreman_rh_cloud/pull/998/files ?

Reply:
This can be reworked how it was before:
```
      <ScalprumProvider
        pluginSDKOptions= {{
          pluginLoaderOptions: {
            transformPluginManifest: manifest => {
            ...
        }}
        api={{
          chrome: {...
        }}
        config={config}
    >
```